pages/juang961225/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>carrusel</title>
+    <link rel="stylesheet" href="../juang961225/style.css">
+</head>
+<body>
+<!--contenedor general-->
+    <main class="main">
+<!--titulo principal-->
+        <h2 class="main__title">word of warcraft class</h2>
+<!--contenedor de las cartas-->
+        <div class="main__container">
+<!--boton izquierdo de las cards-->
+        <button class="main__button">
+            <h2 class="main__container-button"><</h2>
+        </button>
+<!--primera carta-->
+        <div class="main__card">
+            <img class="main__img" src="../juang961225/pictures/paladin.jpg" alt="paladin">
+            <h2 class="main__card-title">paladin</h2>
+            <h3 class="main__content">first content</h3>
```

Comment:
I think you should replace the H3 tags for a list (ul > li) instead because this is more a structural information section. Remember that H's tags are more appropriate for headlines.

Reply:
resolved, thanks for the advice